apps/www/content/docs/dark-mode.mdx

```diff
@@ -23,16 +29,27 @@ export function ThemeProvider({ children, ...props }: ThemeProviderProps) {
 
 ### Wrap your root layout
 
-Add the `ThemeProvider` to your root layout.
+Add the `ThemeProvider` to your root layout, and the two `theme-color` meta tags.
 
-```tsx {1,9-11} title="app/layout.tsx"
+```tsx {1,7-18,20-22} title="app/layout.tsx"
 import { ThemeProvider } from "@/components/theme-provider"
 
 export default function RootLayout({ children }: RootLayoutProps) {
   return (
     <>
       <html lang="en" suppressHydrationWarning>
-        <head />
+        <head>
+          <meta
+            name="theme-color"
+            media="(prefers-color-scheme: light)"
+            content="white"
+          />
+          <meta
+            name="theme-color"
+            media="(prefers-color-scheme: dark)"
+            content="black"
+          />
+        </head>
```

Comment:
Hi Cibi, thanks for your contribution. However it looks like I won't be able to help on this issue anymore: I can't reproduce it. It's probably since a Chrome update.